example/single_operation/get.go

```diff
@@ -45,7 +45,7 @@ func main() {
 	}
 
 	// insert
-	ctx, _ := context.WithTimeout(context.Background(), time.Duration(1000)*time.Millisecond) // 1000ms
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second) // 10s
 	rowKey := []*table.Column{table.NewColumn("c1", int64(1))}
 	insertColumns := []*table.Column{table.NewColumn("c2", int64(2))}
```

Comment:
```
// CREATE TABLE test(c1 bigint, c2 varchar(20), PRIMARY KEY(c1)) PARTITION BY hash(c1) partitions 2;
```
I guess it might conflict with the table schema. `c2` is a string, not a bigint.

Reply:
yes, this is a bug https://github.com/oceanbase/obkv-table-client-go/issues/80, will create new pull request to fix it.